department-of-veterans-affairs/caseflow
# Fixed focus indicator on the schedule veteran page

Resolves [CASEFLOW-1571](https://vajira.max.gov/browse/CASEFLOW-1571)

### Description

Fixes the cutoff focus indicator when tabbing between Hearing Day selections on the Schedule Veteran page

### Acceptance Criteria
- [ ] Visible focus indicator on docket picker is not covered by other elements

### Testing Plan

1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the “Schedule Veteran” button at the top left
4. Select the St. Petersburg, FL RO from the dropdown
5. Tab to the first hearing day and make sure that the focus indicator is fully visible

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="296" alt="Screen Shot 2021-06-28 at 9 50 26 AM" src="https://user-images.githubusercontent.com/61989526/123674288-4b4a0180-d7f6-11eb-8a5a-b14a8e3f908a.png">|<img width="325" alt="HearingDayNavFocusIndicitorAfter" src="https://user-images.githubusercontent.com/61989526/123674300-4e44f200-d7f6-11eb-9e11-5eb5f0775b8d.png">
<img width="1747" alt="Screen Shot 2021-06-28 at 11 52 46 AM" src="https://user-images.githubusercontent.com/61989526/123688875-6d984b00-d807-11eb-9db5-c5c3f6625419.png">|<img width="1683" alt="Screen Shot 2021-06-28 at 11 52 30 AM" src="https://user-images.githubusercontent.com/61989526/123688895-725cff00-d807-11eb-80f9-205c1b42c282.png">
